pydantic/_internal/_fields.py

```diff
@@ -97,6 +99,10 @@ def collect_model_fields(  # noqa: C901
 
     class_vars: set[str] = set()
     for ann_name, ann_type in type_hints.items():
+        # check protected namespaces
+        for p in config_wrapper.protected_namespaces:
```

Comment:
For me the code looks self explanatory. Other checks below don't have similar comments. I'd rather renamed `p` to `protected_namespace` for clarity and removed the comment.